### PR TITLE
add #9 to default rules

### PR DESCRIPTION
**Pull request by xJuvi**
The new rules from #9 are currently not part of the default rules. I added them now and change the new description into alphabetic order.

### CODE REVIEW

File: src/Factory.php [hunk 1]
```diff
@@ -57,6 +57,7 @@ protected function registerDefaultRules()
             'digits'               => new Rules\Digits,
             'digits_between'       => new Rules\DigitsBetween,
             'email'                => new Rules\Email,
+            'end_with'             => new Rules\EndsWith,
```
Comment (dave-redfern) [inline src/Factory.php]: Should be plural: `ends_with`

File: src/Resources/i18n/de.php [hunk 2]
```diff
@@ -22,6 +22,7 @@
     'rule.digits'                 => ':attribute darf nur Ziffern enthalten und muss aus genau :length Ziffern bestehen',
     'rule.digits_between'         => ':attribute darf nur Ziffern enthalten und muss zwischen :min und :max liegen',
     'rule.email'                  => ':attribute ist keine gültige E-Mail-Adresse',
+    'rule.end_with'               => ':attribute muss auf :compare_with enden',
```
Comment (dave-redfern) [inline src/Resources/i18n/de.php]: This should be plural too in both language files.

**Review by dave-redfern — CHANGES_REQUESTED** [review]
Thanks for the corrections!

There are a couple of changes needed, and could you additionally update the `EndsWith` rule as I just noticed it is referencing the starts_with language string.

**Comment by xJuvi** [comment]
Hi @dave-redfern 
i changed like your change request.

Kind regards

File: src/Resources/i18n/en.php [hunk 3]
```diff
@@ -22,6 +22,7 @@
     'rule.digits'                 => ':attribute must be numeric and must have an exact length of :length',
     'rule.digits_between'         => ':attribute be numeric and must have a length between :min and :max',
     'rule.email'                  => ':attribute is not a valid email address',
+    'rule.end_with'               => ':attribute must end with :compare_with',
```
Comment (dave-redfern) [inline src/Resources/i18n/en.php]: Missed this plural 😉

Comment (xJuvi) [reply src/Resources/i18n/en.php]: fixed. Thanks for your hint.

**Review by dave-redfern — APPROVED** [review]

**Pull request merged**